lizrd/text/tokenizers.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
+from transformers import BertTokenizerFast, GPT2TokenizerFast
+
+
+class AbstractTokenizer(ABC):
+    sequence_separator_id: Optional[int]
+    mask_id: Optional[int]
+    eot_id: Optional[int]
+    vocab_size: int
+
+    @abstractmethod
+    def text_to_ids(self, text: str) -> List[int]:
+        raise NotImplementedError()
+
+
+class BertTokenizer(AbstractTokenizer):
+    vocab_size = 30522
```

Comment:
Uppercase maybe?

Reply:
upper case tends to be used more for global variables, right? even if constant, but not global, I'd use lowercase (it's very weak opinion)

Reply:
Kinda on the fence here xd

Reply:
<img width="1004" alt="image" src="https://github.com/llm-random/llm-random/assets/70006947/0edd439b-33fc-4703-9d19-da036319ae26">

Reply:
@crewtool  ma dobre intuicje, polecam tego pana